arlo-client/package.json

```diff
@@ -32,14 +36,17 @@
     "history": "^4.9.0",
     "http-proxy-middleware": "^0.19.1",
     "jspdf": "^1.5.3",
+    "jsdom": "^15.1.1",
+    "qrcode.react": "^0.9.3",
     "react": "16.9.0",
     "react-app-polyfill": "^1.0.1",
-    "react-dom": "16.9.0",
+    "react-dom": "^16.9.0",
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.0.1",
     "react-test-renderer": "^16.8.6",
     "react-toastify": "^5.3.1",
     "styled-components": "^4.3.2",
+    "ts-node": "^8.4.1",
```

Comment:
We no longer need this, right?

Reply:
We might still need it, see above response to comment.